docs/UserGuide.md

```diff
@@ -33,95 +39,184 @@
+Shows a list of all task in the CMDo on that day. It also acts as a filter.<br>
+Format: `find <today>`
+
+> Key in <day> and all task due on that day will appear
+> <br><img src="images/ListTmr.jpeg" width="600"><br>
+> Key in <done> and all done task will appear
+> <br><img src="images/ListDone.jpeg" width="600"><br>
+> Key in any search parameter and list command will add a filter for you
+<br><img src="images/ListTake.jpeg" width="600"><br>
+> Key in </priority> to show all tasks with specified priority
+<br><img src="images/ListHP.jpeg" width="600"><br>
+
+#### Listing all task in CMDo : `list`
+Shows a list of all task in the CMDo.<br>
+Format: `list`
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
+#### Deleting a task in CMDo : `delete`
+Deletes the specified task from the to do list.
 Format: `delete INDEX`
 
 > Deletes the person at the specified `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
-
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
-
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+* `list all`
+  `delete 2`
+  Deletes the 2nd task in the to do list.
+* `find email` 
+  `delete 1`
+  Deletes the 1st task in the results of the `find` command.
 
-Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+#### Edit task : `edit`
+Update existing tasks in the CMDo list, as you would entering a new task.<br>
+Format: `edit INDEX details`  
+
+> Examples: 
+> * `edit 2 eat candy tomorrow`
+>  Edit the 2nd task in the to do list to eat candy with date due as tomorrow 
+>  
+> * `edit 1 Bear`
+> Task 2 changes to Bear
 
-#### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
+
+####  Mark a task as done : `done <index>`
+Done a task so tick it off the list
+
+Format: `done <index>`  
+<br><img src="images/Done.jpeg" width="600"><br>
+<br><img src="images/Done1.jpeg" width="600"><br>
+
+
+#### Undo the last action : `undo`
+Undo the earlier action
+Format: `undo`  
+
+<br><img src="images/Undo.jpeg" width="600"><br>
+<br><img src="images/Undo1.jpeg" width="600"><br>
+
+
+#### Redo earlier action : `redo`
+Redo the earlier action.
+Format: `redo`
+
+#### Change the storage location : `storage`
```

Comment:
do rmb to update ur user guide too if needed along side with ur implmentations~ great job updating the block and find :)